understanding/resize-text.html

```diff
@@ -38,7 +38,7 @@ <h2>意図</h2>
             <p>コンテンツを拡大縮小することは、第一にユーザエージェントが果たすべき役割である。<a href="https://www.w3.org/TR/WAI-USERAGENT/guidelines.html#tech-configure-text-scale">UAAG 1.0 チェックポイント 4.1</a> を満たしているユーザエージェントは、利用者がテキストの拡大縮小を設定できるようにしている。コンテンツ制作者の責任は、ユーザエージェントがコンテンツを効果的に拡大縮小することを妨げないウェブコンテンツを制作することである。コンテンツ制作者は、コンテンツがテキストベースのコントロールを含むテキストのサイズ変更のためのユーザエージェントのサポートを妨げていないことを確認すること、又はテキストのサイズ変更もしくはレイアウトの変更を直接サポートすることによって、この達成基準を満たすことができる。直接サポートの例は、異なるスタイルシートを割り当てるために使用できるサーバーサイドスクリプトを介したものかもしれない。</p>
             <p>利用者が、ズームサポートを持つユーザエージェントにアクセスできない場合、コンテンツ制作者は、HTML コンテンツに対するこの達成基準を満たすためにユーザエージェントに依存することはできない。例えば、IE 6 を使用する必要のある環境で動作する場合などである。</p>
             <div class="note"><div role="heading" class="note-title marker" aria-level="3">訳注</div><p>ほとんどのモダンブラウザでは拡大機能が提供されている。また、IE6 は、サポートが終了したブラウザである。<a href="http://www.atmarkit.co.jp/ait/articles/1503/11/news134.html">いよいよ完全終了へ。Internet Explorer（IE）サポート終了スケジュール：Tech TIPS - ＠IT</a> も参照されたい。</p></div>
-            <p>コンテンツ制作者が、ユーザエージェントによってズームサポートを提供していないウェブコンテンツ技術を使用している場合、コンテンツ制作者はこのタイプの機能を提供するか、又はユーザエージェントによって提供される機能のタイプで動作するコンテンツを提供する義務がある。ユーザエージェントがズーム機能を提供していないが、利用者にテキストの大きさの変更するのを許可する場合、コンテンツ制作者は、テキストの大きさを変更するときにコンテンツが利用可能なままであることを保証する責任がある。</p>
+            <p>コンテンツ制作者がユーザエージェントによってズームサポートを提供していない技術を使用している場合、コンテンツ制作者は、このタイプの機能を直接提供する、又はユーザエージェントによって提供されるタイプの機能と連携するコンテンツを提供する責任がある。ユーザエージェントがズーム機能を提供していないが、利用者にテキストの大きさの変更するのを許可する場合、コンテンツ制作者は、テキストの大きさを変更するときにコンテンツが利用可能なままであることを保証する責任がある。</p>
```

Comment:
「ユーザエージェントによってズームサポートを提供していない」は、日本語として少し微妙な感じがしていまして、以下のいずれかかなと思います。
「ユーザエージェントがズームサポートを提供していない」
「ユーザエージェントによってズームサポートが提供されていない」
「ユーザエージェントによるズームサポートの提供がない」

後半、this type を「このタイプ」、the type を単に「タイプ」としていますが、指しているものは同じかと思いますので両方とも「このタイプ」で良いのではないでしょうか。

```suggestion
            <p>コンテンツ制作者がユーザエージェントによるズームサポートの提供がない技術を使用している場合、コンテンツ制作者は、このタイプの機能を直接提供する、又はユーザエージェントによって提供されるこのタイプの機能と連携するコンテンツを提供する責任がある。ユーザエージェントがズーム機能を提供していないが、利用者にテキストの大きさの変更するのを許可する場合、コンテンツ制作者は、テキストの大きさを変更するときにコンテンツが利用可能なままであることを保証する責任がある。</p>
```